src/client/datascience/ipywidgets/ipyWidgetScriptSource.ts

```diff
@@ -183,6 +183,7 @@ export class IPyWidgetScriptSource implements ILocalResourceUriConverter {
         if (!this.notebook) {
             this.notebook = await this.notebookProvider.getOrCreateNotebook({
                 identity: this.identity,
+                resource: this.identity,
```

Comment:
Why is `resource` the same as `identity`? Or does it not matter in this context?

Reply:
Doesn't matter in this context (basically use the best we have, majority of the time `identify = resoruce`, except in interactive window when you open it without a correspoding py file or when you have multiples.